mlflow/transformers.py

```diff
@@ -1609,13 +1616,19 @@ def _coerce_exploded_dict_to_single_dict(self, data):
             return data
 
     def _strip_input_from_response_in_instruction_pipelines(
-        self, input_data, output, output_key, flavor_config, include_prompt
+        self,
+        input_data,
+        output,
+        output_key,
+        flavor_config,
+        include_prompt=True,
+        remove_newlines=False,
     ):
         """
         Parse the output from instruction pipelines to conform with other text generator
         pipeline types and remove line feed characters and other confusing outputs
         """
-        replacements = {"\n\n": " "}
+        replacements = {"\n\n": " ", "\n": " "}
```

Comment:
I'd use a regular expression here to shrink multiple newline characeters into a space.

Reply:
great point. Updated and added a spacing collapse for a weird edge case that can happen